src/filters/birds.cjs

```diff
@@ -48,7 +48,7 @@ params:
     note: The name of the bird (as used in `author` settings)
 */
 const authorPage = (collection, bird) =>
-  withData(collection, 'data.bird', bird)[0];
+  withData(collection, 'data.bird', bird, true)[0];
```

Comment:
@mirisuzanne This fixed a bug where `david` was returning truthy for bird `davidh`. We may want to check to see if there are other places this is an issue.

Reply:
Fixed in https://github.com/oddbird/oddleventy/pull/245